src/style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: black;
+  padding: 20px;
+}
+
+.head-div {
+  width: 350px;
+  border: 1px solid #e75551;
+  margin-bottom: 40px;
+  padding: 10px;
+  background-color: #308dc0;
+}
+
+.middle {
+  padding: 10px;
+  border: 1px solid #e75551;
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.